protos/controlplane.proto

```diff
@@ -175,6 +175,9 @@ message Sampler {
   // There are known tags that can be used to identify the Sampler type
   // and may be used by the platform to provide additional functionality.
   repeated Tag tags = 7;
+  // Sampler capabilities defines what features can be performed by the
+  // sampler.bool
```

Comment:
`sampler.bool` ?

Reply:
Good catch! Change performed as part of 7730d800706b3b507daead675d56ba7c872f9c34